php-fpm/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apk add --no-cache \
 
 # INSTALL PHP DEPENDENCIES
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin/install-php-extensions
-RUN install-php-extensions intl pcntl pdo_mysql zip apcu opcache redis curl
+RUN install-php-extensions intl pcntl pdo_mysql zip apcu opcache redis curl amqp
```

Comment:
🤔

Reply:
I'm using the amqp library in my only api :P